### ISSUE TITLE
Balance values rounding

### ISSUE THREAD

**Issue by 0xceed** [horizontalsystems/unstoppable-wallet-ios#4598]
Hi everyone. It seems to me, current rounding behaviour doesn't bother anyone besides me, but still it would be nice to have options in settings to apply rounding to balance values like it made in TrustWallet? 

I'm very into finance logging and it is frustrating for me to track exact numbers through blockchain explorers, not to say I need to tap every coin in the main list to see even rounded as is values. 

Also, probably I can do it myself in near future, if no one can work on so low prioritised task. At least I need to know that my PR wouldn't be rejected if I do this. 🙂

TL;DR: In current UI I can not know exact transaction and balance values. Only rounded ones to some point. TrustWallet rounds to closest meaningful number for any currency and that is what I'd like to see.

**Comment by Usernamethoma** [comment]
@0xceed We'd be happy to look into this for you. Kindly file a support ticket here
https://forms.gle/zVGLP7GATPsDHdkH9

**Comment by esengulov** [comment]
@0xceed Hi. to our knowledge Trust also uses some kind of rounding mechanism which is different from ours. 

Should you decide to send a PR we can evaluate but keep in mind that our rounding logic used in many places throughout the app. i.e. market, balance, send, transaction screens :)

**Comment by 0xceed** [comment]
Well, you see, Trust Wallet shows unrounded values, when you tap on account in the main list... And Unstoppable just shows more digits, but still does rounding :)
